js/samples/06.assistants.b.orderBot/src/foodOrderCard.ts

```diff
@@ -0,0 +1,68 @@
+import { Order } from "./foodOrderViewSchema";
+
+export function generateCardForOrder(order: Order) {
+    const card: any = {
+      "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+      "type": "AdaptiveCard",
+      "version": "1.2",
+      "body": [
+        {
+          "type": "TextBlock",
+          "text": "Your Order Summary",
+          "weight": "Bolder",
+          "size": "Medium",
+          "wrap": true
+        },
+        {
+          "type": "Container",
+          "items": [
+            {
+              "type": "TextBlock",
+              "text": "Items:",
+              "weight": "Bolder",
+              "wrap": true
+            }
+          ]
+        }
+      ]
+    };
+
+    for (const item of order.items) {
+        switch (item.itemType) {
+            case 'pizza':
+                const name = item.name ? item.name : 'Custom Pizza';
+                card.body.push({
+                    "type": "TextBlock",
+                    "text": `${name} - Size: ${item.size || 'large'}, Quantity: ${item.quantity || 1}`,
+                    "wrap": true
+                });
+                break;
+            case 'beer':
+                card.body.push({
+                    "type": "TextBlock",
+                    "text": `Beer - Kind: ${item.kind}, Quantity: ${item.quantity || 1}`,
```

Comment:
Culturally, it might be best to leave out alcohol -- currently I'm blanking on the policy name...

Reply:
Milkshake instead?

Reply:
This is a direct port of TypeChat's sample. We can re-work it.